### PR TITLE
TRA update

### PR DESCRIPTION
**Pull request by dianakolusheva**
This PR fixes some issues and makes small adjustments in TRA code related to kappa simulation.

### CODE REVIEW

File: bioagents/tra/tra.py [hunk 1]
```diff
@@ -196,7 +191,7 @@ def plot_results(self, results, agent, obs_name, thresh=50):
         for tspan, yobs in results:
             plt.plot(tspan, yobs[obs_name])
         plt.ylim(-5, max_val_lim)
-        plt.xlim(-100, 10100)
+        plt.xlim(min_time, max_time)
```
Comment (bgyori) [inline bioagents/tra/tra.py]: I think at least the minimum value of `-100` here was on purpose to add a little bit of a gap on the left hand side of the plot, making some trajectories (e.g., ones that go up quickly at the beginning of the simulation) and labels better visible. Is there a situation where `min([result[0][0] for result in results])` would be expected to be something other than 0? If not, then we could keep the fixed `-100` setting or make it adaptive with respect to `max_time`, for instance, by setting it to `-max_time/100`.

**Review by bgyori — COMMENTED** [review]
I think these changes are good, I added one comment. Do we currently have tests for Kappa simulation enabled/running on Github Actions? If not, I think we should add or enable them to make sure these simulation modes are tested.

File: bioagents/tra/tra.py (hunk 1 above)
Comment (dianakolusheva) [reply bioagents/tra/tra.py]: I have an example where the time only ranges between 0 and 3 - having -100 doesn't look good so I changed the range to use `-max_time/100, max_time+max_time/100`

**Comment by dianakolusheva** [comment]
> I think these changes are good, I added one comment. Do we currently have tests for Kappa simulation enabled/running on Github Actions? If not, I think we should add or enable them to make sure these simulation modes are tested.

It looks like there's only one test using Kappa mode - `bioagents.tests.tra_test.TraTestModel1_Kappa`, it only checks that the model can run with Kappa, but doesn't check simulation results.

File: bioagents/tra/tra.py (hunk 1 above)
Comment (bgyori) [reply bioagents/tra/tra.py]: Thanks!

**Review by bgyori — APPROVED** [review]

**Pull request merged**